horovod/spark/common/params.py

```diff
@@ -101,8 +101,10 @@ class EstimatorParams(Params):
                                typeConverter=TypeConverters.toBoolean)
 
     pin_gpu = Param(Params._dummy(), 'pin_gpu',
-                               'Whether to pin the traininig process to the GPU. Defaults to True.',
-                               typeConverter=TypeConverters.toBoolean)
+                    'Whether to pin the traininig process to the GPU. '
+                    'Setting this to False will skipping binding to GPU even when GPU is available.'
+                    'Defaults to True.',
```

Comment:
If not pinning to GPU means not using a GPU at all, then this should be called `use_gpu`. I thought "pin" has a subtle different meaning than "use".

Then, `use_gpu=False` becomes clear what it means even if GPU is available.

I would still keep the warning prints below.

Reply:
i see, then i think we can rename it to use_gpu

Reply:
renamed